packages/client/components/NewMeetingSettingsToggleAnonymity.tsx

```diff
@@ -0,0 +1,86 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {NewMeetingSettingsToggleAnonymity_settings$key} from '~/__generated__/NewMeetingSettingsToggleAnonymity_settings.graphql'
+import useAtmosphere from '../hooks/useAtmosphere'
+import useMutationProps from '../hooks/useMutationProps'
+import SetMeetingSettingsMutation from '../mutations/SetMeetingSettingsMutation'
+import {PALETTE} from '../styles/paletteV3'
+import Checkbox from './Checkbox'
+import PlainButton from './PlainButton/PlainButton'
+
+const ButtonRow = styled(PlainButton)({
```

Comment:
-1 both `NewMeetingSettingsToggleCheckIn` and `NewMeetingSettingsToggleAnonymity` rely on the exact same `ButtonRow`, and the code for the `ButtonRow` component is repeated 2 times in both files, would you consider reusing one component [^1] since they are basically identical?

[^1]: https://github.com/ParabolInc/parabol/commit/73eb08e13d1ba8568567326af964294b2677576c#diff-987709bac0f7c25b6e84f98c2e709d498da60b668cdf85ae9bfe0565ecd09022R13